src/components/Sidebars/IconsSidebar.tsx

```diff
@@ -116,25 +130,29 @@ const IconsSidebar: React.FC<IconsSidebarProps> = ({ openRelativePath, sidebarSh
         onClick={() => setIsNewNoteModalOpen(true)}
       >
         <div className="flex size-4/5 items-center justify-center rounded hover:bg-neutral-700">
-          <VscNewFile className="text-gray-200" size={22} title="New Note" />
+          <IconContext.Provider value={newFileContextValue}>
```

Comment:
Are these ```IconContext.Provider```s necessary? They seem to introduce a lot of baggage code when we could just pass the colour in directly no?

Reply:
Yeah, I think you're right here. I ended up replacing all IconContext.Providers to just color with the appropriate ternaries.